.rubocop.yml

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
There is one unnecessary empty line at the beginning of this file. You can remove it. 😄